build test error

```
$ qmake chunks.pro
$ LC_ALL=C make 
g++ -c -pipe -O2 -std=gnu++11 -Wall -W -D_REENTRANT -fPIC -DMODUL_TEST -DQT_NO_DEBUG -DQT_GUI_LIB -DQT_CORE_LIB -I. -isystem /usr/include/qt5 -isystem /usr/include/qt5/QtGui -isystem /usr/include/qt5/QtCore -I. -I/usr/lib64/qt5/mkspecs/linux-g++ -o testchunks.o testchunks.cpp
testchunks.cpp: In constructor 'TestChunks::TestChunks(QTextStream&, QString, int, bool, int)':
testchunks.cpp:5:92: error: no matching function for call to 'Chunks::Chunks()'
 TestChunks::TestChunks(QTextStream &log, QString tName, int size, bool random, int saveFile)
                                                                                            ^
In file included from testchunks.h:9:0,
                 from testchunks.cpp:1:
../src/chunks.h:35:5: note: candidate: Chunks::Chunks(QIODevice&, QObject*)
     Chunks(QIODevice &ioDevice, QObject *parent);
     ^~~~~~
../src/chunks.h:35:5: note:   candidate expects 2 arguments, 0 provided
../src/chunks.h:34:5: note: candidate: Chunks::Chunks(QObject*)
     Chunks(QObject *parent);
     ^~~~~~
../src/chunks.h:34:5: note:   candidate expects 1 argument, 0 provided
make: *** [Makefile:465: testchunks.o] Error 1

```

--- a/test/testchunks.cpp
+++ b/test/testchunks.cpp
@@ -2,7 +2,7 @@
 #include <cstdlib>
 
 
-TestChunks::TestChunks(QTextStream &log, QString tName, int size, bool random, int saveFile)
+TestChunks::TestChunks(QTextStream &log, QString tName, int size, bool random, int saveFile) : _chunks(nullptr)
 {
     char hex[] = "0123456789abcdef";
     srand(0);

